cli/src/commands/rebase.rs

```diff
@@ -439,42 +609,55 @@ fn rebase_revisions(
                 skipped_commits.iter(),
             )?;
         }
-        let num_rebased = target_commits.len() - skipped_commits.len();
-        if num_rebased > 0 {
-            writeln!(fmt, "Rebased {num_rebased} commits onto destination")?;
+        if num_rebased_targets > 0 {
+            writeln!(
+                fmt,
+                "Rebased {num_rebased_targets} commits onto destination"
+            )?;
         }
-        if num_rebased_descendants > 0 {
+        if num_rebased_others > 0 {
+            // TODO: depending on the destination, this description isn't really accurate.
+            // Should we just say rebased X other commits?
             writeln!(
                 fmt,
-                "Rebased {num_rebased_descendants} descendant commits onto parents of rebased \
-                 commits"
+                "Rebased {num_rebased_others} descendant commits onto parents of rebased commits"
             )?;
         }
```

Comment:
This section is the biggest question mark I have right now, should we just change the message to something like "Rebased X other commits"? Otherwise we probably have to track a lot more state about which commits are being rebased and onto where to provide better status messages.

Reply:
"Rebased X other commits" sounds good to me. Incidentally, I just ran into a similar situation while working on the `jj squash` code. It's not obvious which commits to mention there either. Should we count the destination commit? How about the source commits? Probably not if they were abandoned but maybe if they were rewritten.